package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "myfirstreact",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/jest": "^24.0.18",
```

Comment:
These are development dependencies. Read about what is the difference between production and development dependencies and move them to the development ones.